packages/b/boost/xmake.lua

```diff
@@ -242,9 +252,21 @@ package("boost")
             end
         end
 
+        local function config_deppath(file,depName,ruleName)
+                local dep = package:dep(depName)
+                local info = dep:fetch({external = false})
+                local usStr = format("\nusing %s : : <include>\"%s\" <search>\"%s\" ;",ruleName, info.includedirs[1], info.linkdirs[1])              
```

Comment:
use lowercase, and includedirs may be string instead of table.

Reply:
fix the style and format is done.
includedirs is table with only one element for xz and zstd